controllers/challenge_controller.go

```diff
@@ -0,0 +1,100 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/drift-org/backend/helpers"
+	"github.com/drift-org/backend/models"
+	"github.com/gin-gonic/gin"
+	"github.com/kamva/mgm/v3"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+type ChallengeController interface {
+	Create(context *gin.Context)
+}
+
+type challengeController struct{}
+
+func NewChallengeController() ChallengeController {
+	return &challengeController{}
+}
+
+// TODO: Idea for future, if we want to let companies/orgs create challenges (Business Model #3).
+// Default method of adding challenges on frontend is to add by placing a marker on a map
+// (allowing us to grab Lat/Long coordinates). We can also offer a (premium) option for users to
+// provide just the address, and convert to Lat/Long -- thus allowing batch creation of challenges.
```

Comment:
iirc we were going to implement business model 3 later after we do everything else. Is this route just for our own internal use for now?

Reply:
Yep, for ease of creating challenges. Will probably come in handy for testing later if we have a route for this